src/Aggregate/ConfigurableAggregateTranslator.php

```diff
@@ -59,23 +61,14 @@ class ConfigurableAggregateTranslator implements AggregateTranslator
      */
     private $messageToEventCallback = null;
 
-    /**
-     * @param null|string   $identifierMethodName
-     * @param null|string   $versionMethodName
-     * @param null|string   $popRecordedEventsMethodName
-     * @param null|string   $replayEventsMethodsName
-     * @param null|string   $staticReconstituteFromHistoryMethodName
-     * @param null|callable $eventToMessageCallback
-     * @param null|callable $messageToEventCallback
-     */
     public function __construct(
-        $identifierMethodName = null,
-        $versionMethodName = null,
-        $popRecordedEventsMethodName = null,
-        $replayEventsMethodsName = null,
-        $staticReconstituteFromHistoryMethodName = null,
-        $eventToMessageCallback = null,
-        $messageToEventCallback = null)
+        ?string $identifierMethodName = null,
```

Comment:
When `= null` is provided, `?` is optional. Your decision whether to keep it though